app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes, foreign_key: 'users_id'
```

Comment:
Please add the foreign key `author_id` to make an association between `user and post model`, `user and comment model`, and `user and like model` instead of `user_id` as shown in this [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
Hi @TheLuisAbarca , thanks for reviewing my job and for your kind feedback. The changes are implemented in commit 90ae214.